docs/user-guide/manager-ui/appearance.md

```diff
@@ -4,6 +4,546 @@ sidebar_position: 3
 
 # Appearance Page
 
+## Configuring the Manager UI
+
+The settings option 'Appearance' can be used in a [custom deployment](../deploying/custom-deployment.md). You can use it to style your deployment and configure how the pages of the Manager display the assets and attributes in your system. Some of these settings can be changed visually in the user interface, others can be set when selecting the JSON editor. The configuration gets stored in the manager_config file.
+
+In this user guide we will use an example JSON manager_config and give a short description of each section you could include. Note that some pages have a default config that you can overwrite using the manager_config.
+
+:::note
+
+By default Superusers (e.g. the 'admin' user of the master realm) will see these styling changes. However, if ```"manager":{"applyConfigToAdmin":false}``` is set in the manager_config, most styling changes will _not_ show. This option can be set so that the 'admin' user is not limited by what is set in the config.
```

Comment:
```suggestion
By default Superusers (e.g. the 'admin' user of the master realm) will see these styling changes. However, if `"manager":{"applyConfigToAdmin":false}` is set in the manager_config, most styling changes will _not_ show. This option can be set so that the 'admin' user is not limited by what is set in the config.
```